decidim-admin/app/views/decidim/admin/moderated_users/index.html.erb

```diff
@@ -34,36 +34,76 @@
+            <%= link_to moderation.user.name, decidim.profile_path(moderation.user.nickname) %>
+          </td>
+          <td data-label="<%= t(".nickname") %>">
+            <%= link_to moderation.user.nickname, decidim.profile_path(moderation.user.nickname) %>
+          </td>
         <% else %>
-          <td><%= moderation.user.name %></td>
-          <td><%= moderation.user.nickname %></td>
+          <td data-label="<%= t(".name") %>">
+            <%= moderation.user.name %>
+          </td>
+          <td data-label="<%= t(".nickname") %>">
+            <%= moderation.user.nickname %>
+          </td>
         <% end %>
-        <td>
+
+        <td data-label="<%= t(".reason") %>">
           <% reports = moderation.reports.map { |report| render "report", report: } %>
-          <%= safe_join(reports, ",") %>
+          <%= safe_join(reports, ", ") %>
         </td>
-        <td>
+
+        <td data-label="<%= t("models.moderation.fields.created_at", scope: "decidim.moderations") %>">
           <%= l moderation.created_at, format: :decidim_short %>
         </td>
-        <td>
+
+        <td data-label="<%= t(".reports") %>">
           <%= moderation.report_count %>
         </td>
 
-        <td>
-          <% if !moderation.user.blocked? && allowed_to?(:unreport, :moderate_users) %>
-            <%= icon_link_to "arrow-go-back-line", ignore_moderated_user_path(id: moderation), t(".actions.unreport"), class: "action-icon--unreport", method: :put %>
-          <% end %>
-          <% if allowed_to?(:block, :moderate_users) %>
-            <% if moderation.user.blocked? %>
-              <%= icon_link_to "refresh-line", user_block_path(user_id: moderation.user.id), t(".actions.unblock"), class: "action-icon action-icon--disabled", method: :delete %>
-            <% else %>
-              <%= icon_link_to "lock-2-line", new_user_block_path(user_id: moderation.user.id), t(".actions.block"), class: "action-icon action-icon" %>
-            <% end %>
-          <% end %>
+        <td class="table-list__actions" data-label="<%= t(".actions.title") %>">
+          <button type="button" data-component="dropdown" data-target="actions-user-<%= moderation.user.id %>" aria-label="<%= t("decidim.admin.actions.actions_label", resource: moderation.user.nickname) %>">
+            <%= icon "more-fill", class: "text-secondary" %>
+          </button>
+
+          <div class="inline-block relative">
+            <ul id="actions-user-<%= moderation.user.id %>" class="dropdown dropdown__action" aria-hidden="true">
+              <% if !moderation.user.blocked? && allowed_to?(:unreport, :moderate_users) %>
+                <li class="dropdown__item">
+                  <%= link_to ignore_moderated_user_path(id: moderation), method: :put, class: "dropdown__button" do %>
+                    <%= icon "arrow-go-back-line" %>
+                    <%= t(".actions.unreport") %>
+                  <% end %>
+                </li>
+
+                <hr>
```

Comment:
This may be wrongly displayed if the condittion from line 77 is true and the condition from line 88 is false